src/components/Firestore/index.tsx

```diff
@@ -152,8 +155,7 @@ export const Firestore: React.FC<React.PropsWithChildren<unknown>> = React.memo(
               </div>
             }
           >
-            The Emulator Suite UI only supports the (default) database right
-            now.
+            The Emulator Suite UI supports multiple DBs by editing the database name in the URL.
```

Comment:
Totally reasonable for now, but this does seem like low hanging fruit to provide a UI component to choose from a list of existing DBs or 'create' a new one by typing in a databaseId and redirecting to the approriate url.

Reply:
100% we should do this - all it takes it a dropdown or something